debian/control.in

```diff
@@ -45,10 +45,9 @@ Build-Depends: acl-dev,
  ncurses-dev,
  pkg-config,
  po-debconf (>= 0.8.2),
- python-dev,
+# python-dev,
  zlib1g-dev,
  systemd,
- dh-systemd,
```

Comment:
maybe we should tag it with the versions where we want it like this:
```
dh-systemd <buster> <stretch> <xenial> <bionic>,
```
This way at least our own builds will require it for these older releases.

Reply:
Good point